app/components/chart/footer.js

```diff
@@ -41,82 +41,40 @@ var TidelineFooter = React.createClass({
       'patient-data-subnav-hidden': this.props.chartType === 'no-data'
     });
 
-    function getValuesLinkText(props) {
-      if (props.chartType === 'weekly') {
-        if (props.showingValues) {
-          return 'Hide numbers';
-        }
-        else {
-          return 'Show numbers';
-        }
-      }
-      else {
-        return '';
-      }
-    }
-
-    function getLinesLinkText(props) {
-      if (props.chartType === 'modal') {
-        if (props.showingLines) {
-          return 'Hide lines';
-        }
-        else {
-          return 'Show lines';
-        }
-      }
-      else {
-        return '';
-      }
-    }
-
-    function getGroupLinkText(props) {
-      if (props.chartType === 'modal') {
-        if (props.grouped) {
-          return 'Ungroup';
-        }
-        else {
-          return 'Group';
-        }
-      }
-      else {
-        return '';
-      }
-    }
-
-    function getOverlayLinkText(props) {
-      if (props.chartType === 'modal') {
-        if (props.boxOverlay) {
-          return 'Hide range & average';
-        }
-        else {
-          return 'Show range & average';
-        }
-      }
-      else {
-        return '';
-      }
-    }
-
-    var valuesLinkText = getValuesLinkText(this.props);
-
-    var linesLinkText = getLinesLinkText(this.props);
-
-    var groupLinkText = getGroupLinkText(this.props);
-
-    var overlayLinkText = getOverlayLinkText(this.props);
-
     /* jshint ignore:start */
     var showValues = (
-      <a href="" onClick={this.props.onClickValues}>{valuesLinkText}</a>
+      <label htmlFor="valuesCheckbox" ref="label">
+        <input type="checkbox" name="valuesCheckbox" id="valuesCheckbox"
+          checked={this.props.showingValues}
+          onChange={this.props.onClickValues}
+          ref="control" /> Values
```

Comment:
The 'label' and 'control' refs aren't being used that I can see, and React refs have to be unique, I believe (and the same ones appear on all the label/input pairs). Maybe just omit them?